samples/drivers/w1/scanner/app.overlay

```diff
@@ -0,0 +1,7 @@
+&arduino_i2c {
+	w1: w1@18 {
+		compatible = "maxim,ds248x";
+		reg = <0x18>;
+		label = "W1";
```

Comment:
remove `label` is shouldn't be needed.

Reply:
`maxim,ds248x.yaml` includes `i2c-device.yaml` which requires it.

Reply:
ah, ok for now.